pkgs/by-name/du/dummyhttp/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dummyhttp";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "svenstaro";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-MDkyJAVNpCaAomAEweYrQeQWtil8bYo34ko9rAu+VBU=";
+  };
+
```

I suggest adding `darwin.apple_sdk.frameworks.Security` to buildInputs to build it in *-darwin. It failed to build without Security in aarch64-darwin, but I added the following then it succeeded.

```suggestion

  buildInputs = lib.optionals stdenv.isDarwin [
    darwin.apple_sdk.frameworks.Security
  ];

```